.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
-* @hors @egegunes @inelpandzic @pooknull
+* @hors @egegunes @inelpandzic @pooknull @gkech
 /e2e-tests/ @tplavcic @nmarukovich @ptankov @jvpasinatto @eleo007
 Jenkinsfile @tplavcic @nmarukovich @ptankov @jvpasinatto @eleo007
```

Comment:
could you please delete tplavcic from this file, he's no longer in our team

Reply:
PTAL